ruby.c

```diff
@@ -403,7 +404,10 @@ usage(const char *name, int help, int highlight, int columns)
     unsigned int w = (columns > 80 ? (columns - 79) / 2 : 0) + 16;
 #define SHOW(m) show_usage_line(&(m), help, highlight, w, columns)
 
-    printf("%sUsage:%s %s [options] [--] [filepath] [arguments]\n", sb, se, name);
+    printf("%sUsage:%s %s [options] [--] [filepath] [arguments]\n\n", sb, se, name);
+    printf("Details and examples at https://docs.ruby-lang.org/en/%s/ruby/options_md.html\n",
+           RUBY_PATCHLEVEL == -1 ? "master" : STRINGIZE(RUBY_VERSION_MAJOR) "." STRINGIZE(RUBY_VERSION_MINOR));
+
```

Comment:
Detailed info should be only when `--help`.
And keep revision info only in version.c.